src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1547,6 +1547,70 @@ hibernate.connection.url=jdbc:hsqldb:mem:rm;hsqldb.tx=mvcc;hsqldb.lob_file_scale
 
 To switch to a standalone database, see <<_perf_tuning_db>>.
 
+== Nodes and Task Recovery
+
+While your tasks are running on nodes, the scheduler can experience a transient failure and come back to life without
+losing the on-going computations. This is possible thanks to the nodes and task recovery mechanism run by the scheduler.
+When the scheduler is starting, it checks the nodes and the tasks that were persisted in database to be able to recover
+on going tasks and alive nodes. For each node that is supposed to be handled by the scheduler, the recovery will consist
+into trying to contact these nodes. If a node is alive, the reference that is retrieved with the lookup of this node is
```

Comment:
As an introduction paragraph, it goes too much into details.  It should be a few sentences describing, in general, what Nodes and Task Recovery means